### PR TITLE
chore: Add docs for creating a personal access token

### PR DESCRIPTION
**Pull request by RobertGemmaJr**
We've discovered that you have to create a personal access token in order to run `npm install` when cloning the honeycomb repository. The documentation now walks through how to do so, as well as information on the git workflow

### CODE REVIEW

**Comment by RobertGemmaJr** [comment]
I made sure to also add a section about deleting `.npmrc` from git so people's keys don't end up on github

File: docs/quick_start.md [hunk 1]
```diff
@@ -78,20 +101,56 @@ Install [Clang](https://clang.llvm.org/get_started.html) or follow installation
...
+- Paste the code after the = from before and save. It should look as such:
+
+    ```
+    //npm.pkg.github.com/:_authToken=ghp_abcde12345
+    ```
+
+`.npmrc` now contains a secret key, we must tell git to stop tracking it.
+
+- Add ".npmrc" to the end of the `.gitignore` file
+- Remove the file from git:
```
Comment (eldu) [inline docs/quick_start.md]: Maybe we should add `.npmrc` to our gitignore and not rely on users doing it.

Set the current `.npmrc` file in this repo to `.npmrc.example` and add the stub for this permission key.

Then tell the user to copy the file. And fill it out with their key
```
cp .npmrc.example .npmrc
```

This is something we do for the soma backend repo as well. Let me know what you think !

**Review by eldu — COMMENTED** [review]
Interesting! I thought this is what we ended up doing yesterday! Was it not?

Anyway, I added in a idea let me know what you think! I'll approve this PR either way just wanted to hear your thoughts first!

**Review by jashlu — APPROVED** [review]
Look's great, directions and screenshots are helpful, worked for me to get the app running

File: docs/quick_start.md (hunk 1 above)
Comment (RobertGemmaJr) [reply docs/quick_start.md]: Windows can be annoying with file extensions but if people are already in their IDE at this point I think that's a great idea!

Comment (eldu) [reply docs/quick_start.md]: ooh good point

**Review by eldu — APPROVED** [review]
Thanks for adding this to the docs!

**Comment by RobertGemmaJr** [comment]
I'm going to add some information about submitting a pull request before merging into main then I'll close this PR

File: docs/quick_start.md [hunk 2]
```diff
@@ -26,7 +25,29 @@ cd your-new-task-name
 
 ### 2. Change name and description
 
-Update the `package.json` fields to reflect your app name and description, e.g. `name`, `author`, `repository`
+It's best practice to create a new branch whenever we look to make changes and/or add a new feature. Your IDE may have a way to do this or it can always be done on the command line.
+
+```
+git switch -c your-new-branch-name
```
Comment (mirestrepo) [inline docs/quick_start.md]: Does this command create the branch if it doesn't exist?

I always do 

`git checkout -b <branch-name>` to Create a new branch and switch to it.

Comment (mirestrepo) [reply docs/quick_start.md]: I guess @AisOmar could mentioned the different ways to create/switch and touch on the nuances of when the branch starts blank, based on HEAD and so on

File: docs/quick_start.md [hunk 3]
```diff
@@ -26,7 +25,29 @@ cd your-new-task-name
...
+Open `package.json` and edit it to reflect your app name and description (e.g. `name`, `author`, `repository`). Save your changes and commit them to git:
+
+```
+git commit -m "Commit message goes here!"
+```
+
+Now you can switch back to your main branch and merge the changes in!
+
+```
+git switch main
```
Comment (mirestrepo) [inline docs/quick_start.md]: Yes, would push and open a PR instead of merging to `main`

File: docs/quick_start.md (hunk 1 above)
Comment (mirestrepo) [reply docs/quick_start.md]: I like this strategy. But if we are asking to add tokens to it, we should definitely have it in the `.gitignore`

File: docs/quick_start.md [hunk 4]
```diff
@@ -78,20 +101,56 @@ Install [Clang](https://clang.llvm.org/get_started.html) or follow installation
...
+    //npm.pkg.github.com/:_authToken=ghp_abcde12345
+    ```
+
+`.npmrc` now contains a secret key, we must tell git to stop tracking it.
+
+- Add ".npmrc" to the end of the `.gitignore` file
+- Remove the file from git:
+
+    ```
+    git rm --cached .npmrc
```
Comment (mirestrepo) [inline docs/quick_start.md]: I can remove this one from the repo

**Review by mirestrepo — CHANGES_REQUESTED** [review]
This is great. Maybe lets adjust the `.npmrc` so we have the the example file, and the `.npmrc` removed by default. And lets adjust the direct merge into `main`

File: docs/quick_start.md (hunk 2 above)
Comment (AisOmar) [reply docs/quick_start.md]: @mirestrepo Can do!

File: docs/quick_start.md (hunk 4 above)
Comment (RobertGemmaJr) [reply docs/quick_start.md]: Added it on a PR! https://github.com/brown-ccv/honeycomb/pull/106

File: docs/quick_start.md (hunk 2 above)
Comment (RobertGemmaJr) [reply docs/quick_start.md]: Yeah it's the same basic idea but (from what I've seen), I'll change the language to git checkout

**Comment by benjamin-heasly** [comment]
This looks really helpful!

I think the dependency that's needed from the GitHub package registry is CCV behavioral-task-trials 2.0.  I'm wondering if we can sidestep the whole issue by getting the required version of behavioral-task-trials over to the public npmjs registry.  This might could simplify the process here back to plain old `npm install`, and should also make the CI actions "just work".

[This issue](https://github.com/brown-ccv/behavioral-task-trials/issues/29) is encouraging, suggesting there is/was a process for publishing behavioral-task-trials to the public registry?  Unfortunately, I don't any details of how that was set up.

**Review by benjamin-heasly — APPROVED** [review]
I hope this can become simpler once behavioral-task-trials is in the public registry.  Until then, this should be really helpful!

**Comment by RobertGemmaJr** [comment]
Yes! Behavioral Tasks in now in the public NPM registry! Everyone can just run `npm install` and it will work 😄

**Review by eldu — APPROVED** [review]
🚢

**Review by mirestrepo — APPROVED** [review]

**Pull request merged**